climada/hazard/base.py

```diff
@@ -788,7 +797,7 @@ def maybe_repeat(values: np.ndarray, times: int) -> np.ndarray:
                 ],
                 # The accessor for the data in the Dataset
                 accessor=[
-                    lambda x: to_csr_matrix(default_accessor(x)),
+                    to_csr_matrix,
                     lambda x: maybe_repeat(default_accessor(x), num_events),
                     lambda x: strict_positive_int_accessor(x),
```

Comment:
@emanuel-schmid The superfluous lambda caused a `NORMAL` linter warning: `unnecessary-lambda` 😬

Reply:
pylint is a philistine!